### DISCUSSION TITLE
Can handle markdown?

### DISCUSSION THREAD

**Discussion by bilogic** [joshembling/laragenie#17] [Q&A]
Can this handle the human language in markdown? e.g. a knowledgebase etc, or is there some special configuration at openai side?

**Comment by joshembling** [comment]
Hi @bilogic 

What is your use case?

**Reply by bilogic** [reply]
Hi thanks for getting back, to answer common customer questions based on FAQs etc

**Reply by joshembling** [reply]
Yes, you can input in any format you would like, including markdown - and recieve the results in markdown.
